examples/jsm/shaders/FilmShader.js

```diff
@@ -38,7 +38,7 @@ const FilmShader = {
 
 			vec4 base = texture2D( tDiffuse, vUv );
 
-			float noise = rand( vUv + time );
+			float noise = rand( vUv + mod(time,1.0) );
```

Comment:
> rand() coment:
> expects values in the range of [0,1]x[0,1]

This will still pass results in that are > 1. The modulus should wrap `vUv + time`.